app/services/FinancialInstitutionsService.scala

```diff
@@ -173,11 +173,10 @@ class FinancialInstitutionsService @Inject() (connector: FinancialInstitutionsCo
 
   private def extractAddress(userAnswers: UserAnswers): Option[AddressDetails] =
     userAnswers
-      .get(FetchedRegisteredAddressPage)
-      .map(_.toAddress)
-      .orElse(userAnswers.get(UkAddressPage))
+      .get(UkAddressPage)
       .orElse(userAnswers.get(NonUkAddressPage))
```

Comment:
hey @matthewharris1  as per Jill comment, we shouldn't be expecting to create any FI for non UK address